### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by yanko** [Cthutu/oogl#1]
DO you have any examples on how to use?

**Comment by Cthutu** [comment]
It was written as support for another piece of code.  It is not finished
and requires use with other libraries.  I have largely abandon it due to
the fact that I am switching permanently to another programming language
and will redesign it to be more declarative.  You're welcome to take the
code and reuse it how you see fit.  My Xu library (a window manager, also
unfinished) uses it xu-window.cpp and xu-panel.cpp.

Thanks.  Sorry it isn't more complete.

**Comment by yanko** [comment]
alright, tx.
will you be sharing this other redesigned version in the new lang?

**Comment by Cthutu** [comment]
Yes. But I'll be writing the compiler for the new language first. I'm
starting a dev project I've been putting off for years. But I might write
the new open gl library before that in C actually
